docs/source/content.rst

```diff
@@ -5,14 +5,14 @@ plone.restapi does not only expose content objects via a RESTful API. The API co
 
 Manipulating resources across the network by using HTTP as an application protocol is one of core principles of the REST architectural pattern. This allows us to interact with a specific resource in a standardized way:
 
-======= ======================= ==============================================
-Verb    URL                     Action
-======= ======================= ==============================================
-POST    /folder                 Creates a new document within the folder
-GET     /folder/{document-id}   Request the current state of the document
-PATCH   /folder/{document-id}   Update the document details
-DELETE  /folder/{document-id}   Remove the document
-======= ======================= ==============================================
+======= =============================    ==============================================
+Verb    URL                              Action
+======= =============================    ==============================================
+POST    /{{folder_name}}                 Creates a new document within the folder
+GET     /{{folder_name}}/{document-id}   Request the current state of the document
```

Comment:
@prakharjoshi we should either use single brackets or double brackets. Though, we should not mix both.